code/modules/games/tarot.dm

```diff
@@ -41,3 +41,59 @@
 		cooldown = world.time
 	else
 		return
+
+///Fluff item, using a separate item rather than a subitem to customise New()
+
+/obj/item/deck/dark_tarot
+	name = "dark rose tarot deck"
+	desc = "A limited edition tarot deck for the scene-girl inclined!"
+	icon_state = "dark_tarot"
+
+/obj/item/deck/dark_tarot/New()
+	..()
+
+	var/datum/playingcard/P
+	for(var/name in list("fool","magician","high priestess","empress","emperor","hierophant","lovers","chariot","strength","hermit","wheel of fortune","justice","hanged man","death","temperance","devil","tower","star","moon","sun","judgement","world","white dragon with blue eyes","charred lizard","dark lotus","bash","reverse","rules","acid","abyss","maw"))
+		P = new()
+		P.name = "[name]"
+		if(name == "high priestess")
+			P.card_icon = "dark_high_priestess"
+		else if(name == "wheel of fortune")
+			P.card_icon = "dark_wheel"
+		else if(name == "hanged man")
+			P.card_icon = "dark_hanged"
+		else if(name == "white dragon with blue eyes")
+			P.card_icon = "dark_dragon"
+		else if(name == "charred lizard")
+			P.card_icon = "dark_lizard"
+		else if(name == "dark lotus")
+			P.card_icon = "dark_lotus"
+		else
+			P.card_icon = "dark_[name]"
+		P.back_icon = "dark_back_tarot"
+		cards += P
+	for(var/suit in list("wands","pentacles","cups","swords"))
+		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten","page","knight","queen","king"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "dark_[suit]"
+			P.back_icon = "dark_back_tarot"
+			cards += P
+
+/obj/item/deck/dark_tarot/shuffle()
+	var/mob/living/user = usr
```

Comment:
horrible xS, the shuffle proc should get mob/user 

and in card, the verb should call shuffle with usr

Reply:
I agree! Though that's a bit out of the scope of this one maybe, because this is just a variant of the normal tarot deck above it (and playing cards in general). After this one is merged, I can go through and fix up all those rather than doing it as part of a fluff item.

Reply:
Or maybe I'll do it all in one once it's been approved actually.

Reply:
Can you check I've done it as you meant?